auto_nag/bugbug_utils.py

```diff
@@ -97,10 +64,31 @@ def get_bugs(self, date="today", bug_ids=[]):
 
         bugs = self.remove_using_history(bugs)
 
+        # Recreate bug ids as some of the bugs might have been filtered out
+        bug_ids = [bug["id"] for bug in bugs]
+
         # Analyze bugs (make a copy as bugbug could change some properties of the objects).
         if len(bug_ids) > 0:
-            probs = self.model.classify(copy.deepcopy(bugs), True)
+            url = f"{BUGBUG_HTTP_SERVER}/{model}/predict/batch"
+
+            for i in range(100):
+                response = requests.post(
+                    url, headers={"X-Api-Key": "Test"}, json={"bugs": bug_ids}
+                )
+                if response.status_code == 200:
+                    break
+                elif response.status_code == 202:
+                    # All the results are not ready yet, try again in 1 second
+                    time.sleep(1)
+                else:
+                    response.raise_for_status()
+
```

Comment:
We should do something when we run out of retries

Reply:
I've added a basic exception in https://github.com/mozilla/relman-auto-nag/pull/733/commits/5fa78296240c5f2e34421444768759482b95c911, there is an example output in the commit message.